generators/server/templates/dotnetcore/src/Project/Program.cs.ejs

```diff
@@ -66,11 +73,27 @@ namespace <%= namespace %> {
         private static ILogger CreateLogger()
         {
             var appConfiguration = GetAppConfiguration();
+            var port = 6514;
 
             // for logger configuration
             // https://github.com/serilog/serilog-settings-configuration
+            if (appConfiguration.GetSection(SerilogSection)[SyslogPort] != null)
+            {
+                if(int.TryParse(appConfiguration.GetSection(SerilogSection)[SyslogPort], out var portFromConf))
+                {
+                    port = portFromConf; 
+                }
+            }
+
+            var url = appConfiguration.GetSection(SerilogSection)[SyslogUrl] != null
+                ? appConfiguration.GetSection(SerilogSection)[SyslogUrl]
+                : "localhost";
+            var appName = appConfiguration.GetSection(SerilogSection)[SyslogAppName] != null
+                ? appConfiguration.GetSection(SerilogSection)[SyslogAppName]
+                : "JhipsterApp";
             var loggerConfiguration = new LoggerConfiguration()
                 .Enrich.With<LoggerNameEnricher>()
+                .WriteTo.TcpSyslog("localhost",6514,"testappserilog", FramingType.OCTET_COUNTING,SyslogFormat.RFC5424,Facility.Local0,SslProtocols.None, new CertificateFileProvider(certFilename, String.Empty),CertValidationCallback)
```

Comment:
You could use the variable Line 76

Reply:
Yes, i do this but i revert with bad commit, i fix it